command/kv_patch.go

```diff
@@ -264,6 +264,11 @@ func (c *KVPatchCommand) Run(args []string) int {
 		return PrintRawField(c.UI, secret, c.flagField)
 	}
 
+	// If the secret is wrapped, return the wrapped response.
+	if secret != nil && secret.WrapInfo != nil && secret.WrapInfo.TTL != 0 {
```

Comment:
this is a bit of a nit, but I noticed that we already check if `secret` is nil above, and return from the function if that's true, so I don't think we need to check once again if it's nil

Reply:
Agreed

Reply:
```suggestion
	if secret.WrapInfo != nil && secret.WrapInfo.TTL != 0 {
```

Reply:
Thanks!